projects/boringssl/build.sh

```diff
@@ -40,8 +40,9 @@ for F in $fuzzerFiles; do
   fuzzerName=$(basename $F .cc)
   echo "Building fuzzer $fuzzerName"
   $CXX $CXXFLAGS \
+      -D_BORINGSSL_LIBPKI_ -std=c++1z \
       -o $OUT/${fuzzerName} $LIB_FUZZING_ENGINE $F \
-      -I $SRC/boringssl/include ./ssl/libssl.a  ./crypto/libcrypto.a
+      -I $SRC/boringssl/include ./ssl/libssl.a  ./crypto/libcrypto.a ./libpki.a
```

Comment:
CI seems to be failing. I think you may need to put `libpki.a` before `libcrypto.a`

Also, ugh, this is annoying. We should fix the oss-fuzz build so the fuzzer is built by CMake. Probably we need a new CMake flag because `FUZZ` enables some extra things.

Reply:
reordered them, so hopefully this builds.